packages/react-core/src/components/TextArea/examples/TextArea.md

```diff
@@ -223,6 +223,36 @@ import { TextArea } from '@patternfly/react-core';
 <TextArea aria-label="disabled text area example" isDisabled />;
 ```
 
+### Read only
+
+```js
+import React from 'react';
+import { Checkbox, TextArea } from '@patternfly/react-core';
+
+const ReadOnlyTextArea = () => {
+  const [isPlainChecked, setIsPlainChecked] = React.useState(false);
+
+  return (
+    <React.Fragment>
+      <div style={{ marginBottom: '12px' }}>
+        <Checkbox
+          id="isPlain"
+          key="isPlain"
+          label="isPlain"
```

Comment:
```suggestion
          label="Plain read only variant"
```
Thinking that `isPlain` as the label may make it seem like an `isPlain` prop has to be passed in. If updated this would also apply to the text input example file as well.

Reply:
Good call - updated!

Reply:
Good call - updated!